add-ons/prometheus/README.md

```diff
@@ -1,7 +1,10 @@
 # Prometheus Add-on
 An addon that will deploy Prometheus.
 
-NOTE: Requires Origin >= 3.6.0-rc.0
+## Prerequisites
+
+- Origin version >= 3.6.0-rc.0
+- [`admin-user`](https://github.com/minishift/minishift/tree/master/addons/admin-user) add-on should be available
```

Comment:
"should be available"?  I guess you mean "the default add-on [`admin-user`](https://github.com/minishift/minishift/tree/master/addons/admin-user) needs to be enabled"

Reply:
Done